### PR TITLE
[Doc] Add Chinese version of getting start doc

### PR DESCRIPTION
**Pull request by BIGWangYuDong**
This PR adds a Chinese version of getting_start.

Meanwhile, this PR also called on the community to produce Chinese documents. (Adding News in README.md and READ_zh_CN.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,11 +42,11 @@ Apart from MMDetection, we also released a library [mmcv](https://github.com/ope
 
 This project is released under the [Apache 2.0 license](LICENSE).
 
-## Changelog
+## News
```
Comment (hhaAndroid) [inline README.md:45]: Do Changelog records need to be deleted?

File: README_zh-CN.md [hunk 2]
```diff
@@ -42,11 +42,11 @@ v1.x 的历史版本支持 PyTorch 1.1 到 1.4，但是我们强烈建议用户
 
 该项目采用 [Apache 2.0 开源许可证](LICENSE)。
 
-## 更新日志
+## 新闻
 
-最新的月度版本 v2.13.0 在 2021.06.01 发布。
-如果想了解更多版本更新细节和历史信息，请阅读[更新日志](docs/changelog.md)。
-在[兼容性说明文档](docs/compatibility.md)中我们提供了 1.x 和 2.0 版本的详细比较。
+我们正在制作 MMDetection 的中文文档和中文教程，他们都是基于[英文文档](docs/)进行翻译的，文档的框架已经做好并放在了[这里](docs_zh-CN/)，欢迎大家将英文文档翻译成中文，具体的提交要求请参考[这里](https://openmmlab.feishu.cn/docs/doccnhOhfcRrgD8aAvrVfs6DuOe)，希望你也能为 MMDetection 做出贡献。
```
Comment (hhaAndroid) [inline README_zh-CN.md:47]: [英文文档] space

File: docs_zh-CN/get_started.md [hunk 3]
```diff
@@ -1 +1,229 @@
...
+- CUDA 9.2+ （如果基于PyTorch源码安装，也能够支持CUDA9.0）
+- GCC 5+
+- [MMCV](https://mmcv.readthedocs.io/en/latest/#installation)
+
+MMDetection 和 MMCV 版本兼容性如下所示，需要安装正确的 MMCV 版本以避免安装出现问题。
+
+| MMDetection 版本 |        MMCV 版本         |
+| :--------------: | :----------------------: |
+|      master      | mmcv-full>=1.3.3, <1.4.0 |
+|      2.13.0      | mmcv-full>=1.3.3, <1.4.0 |
```
Comment (hhaAndroid) [inline docs_zh-CN/get_started.md:15]: 2.14.0

File: docs_zh-CN/get_started.md [hunk 4]
```diff
@@ -1 +1,229 @@
...
+假设当前已经成功安装 CUDA 10.1，这里提供了一个完整的基于 conda 安装 MMDetection 的脚本：
+
+```shell
+conda create -n open-mmlab python=3.7 -y
+conda activate open-mmlab
+
+conda install pytorch==1.6.0 torchvision==0.7.0 cudatoolkit=10.1 -c pytorch -y
+
+# 安装最新版本的 mmcv
+pip install mmcv-full==latest+torch1.6.0+cu101 -f https://download.openmmlab.com/mmcv/dist/index.html
```
Comment (hhaAndroid) [inline docs_zh-CN/get_started.md:192]: This writing method is currently not supported and will report an error.

File: docs_zh-CN/get_started.md [hunk 5]
```diff
@@ -1 +1,229 @@
...
+cd mmdetection
+pip install -r requirements/build.txt
+pip install -v -e .
+```
+
+### 使用多个 MMDetection 版本进行开发
+
+训练和测试的脚本已经在 PYTHONPATH 中进行了修改，以确保脚本使用当前目录中的 MMDetection。
+
+要使环境中安装默认的 MMDetection 而不是当前重在使用的，可以删除出现在相关脚本中的代码：
```
Comment (hhaAndroid) [inline docs_zh-CN/get_started.md:205]: 重?

**Comment by ZwwWayne** [comment]
Need to resolve conflicts.

File: docs_zh-CN/get_started.md [hunk 6]
```diff
@@ -1 +1,229 @@
...
+
+(1) 按照上述说明，MMDetection 安装在 `dev` 模式下，因此在本地对代码做的任何修改都会生效，无需重新安装；
+
+(2) 如果希望使用 `opencv-python-headless` 而不是 `opencv-python`， 可以在安装 MMCV 之前安装；
+
+(3) 一些安装依赖是可以选择的。例如只需要安装最低运行要求的版本，则可以使用 `pip install -v -e .` 命令。如果希望使用可选择的像 `albumentations` 和 `imagecorruptions` 这种依赖项，可以使用 `pip install -r requirements/optional.txt ` 进行手动安装，或者在使用 `pip` 时指定所需的附加功能（例如 `pip install -v -e .[optional]`），支持附加功能的有效键值包括  `all`、`tests`、`build` 以及 `optional` 。
+
+### 只在CPU安装
+
+我们的代码能够建立只使用 CPU 的环境（CUDA 不可用）。
```
Comment (hhaAndroid) [inline docs_zh-CN/get_started.md:138]: 建立在

File: docs_zh-CN/get_started.md [hunk 7]
```diff
@@ -1 +1,229 @@
...
+
+**注意：**
+
+(1) 按照上述说明，MMDetection 安装在 `dev` 模式下，因此在本地对代码做的任何修改都会生效，无需重新安装；
+
+(2) 如果希望使用 `opencv-python-headless` 而不是 `opencv-python`， 可以在安装 MMCV 之前安装；
+
+(3) 一些安装依赖是可以选择的。例如只需要安装最低运行要求的版本，则可以使用 `pip install -v -e .` 命令。如果希望使用可选择的像 `albumentations` 和 `imagecorruptions` 这种依赖项，可以使用 `pip install -r requirements/optional.txt ` 进行手动安装，或者在使用 `pip` 时指定所需的附加功能（例如 `pip install -v -e .[optional]`），支持附加功能的有效键值包括  `all`、`tests`、`build` 以及 `optional` 。
+
+### 只在CPU安装
```
Comment (hhaAndroid) [inline docs_zh-CN/get_started.md:136]: space

File: docs_zh-CN/get_started.md [hunk 8]
```diff
@@ -1 +1,229 @@
...
+
+我们的代码能够建立只使用 CPU 的环境（CUDA 不可用）。
+
+在CPU模式下，可以运行 `demo/webcam_demo.py` 示例，然而以下功能将在CPU模式下不能使用：
```
Comment (hhaAndroid) [inline docs_zh-CN/get_started.md:140]: space

File: docs_zh-CN/get_started.md [hunk 9]
```diff
@@ -1 +1,229 @@
...
+因此，如果尝试使用包含上述操作的模型进行推理，将会报错。下表列出了由于依赖上述算子而无法在 CPU 上运行的相关模型：
+
+|                        Operator                         |                            Model                             |
+| :-----------------------------------------------------: | :----------------------------------------------------------: |
+| Deformable Convolution/Modulated Deformable Convolution | DCN、Guided Anchoring、RepPoints、CentripetalNet、VFNet、CascadeRPN、NAS-FCOS、DetectoRS |
+|                      MaskedConv2d                       |                       Guided Anchoring                       |
+|                         CARAFE                          |                            CARAFE                            |
+|                      SyncBatchNorm                      |                           ResNeSt                            |
+
+**注意**： MMDetection 目前不支持使用CPU进行训练。
```
Comment (hhaAndroid) [inline docs_zh-CN/get_started.md:164]: space

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**